src/components/Navbar/Navbar.css

```diff
@@ -46,7 +46,7 @@
   padding: 0;
 }
 
-.login-btn {
+.hide-on-mobile {
   display: none;
 }
 
```

Comment:
try a better naming convention here

Reply:
Yes Nikhil, this naming is based on breakpoints.